src/fio/fioEngine.js

```diff
@@ -47,8 +38,28 @@ export class FioEngine extends CurrencyEngine {
     super(currencyPlugin, walletInfo, opts)
     this.fetchJson = fetchJson
     this.fioPlugin = currencyPlugin
-    this.activatedAccountsCache = {}
-    this.otherMethods = {}
+    this.fioSDK = new FIOSDK(
+      walletInfo.keys.fioKey,
+      walletInfo.keys.publicKey,
+      API_URL,
+      this.fetchJson
+    )
+    this.dirtyData = this.localDataDirty.bind(this)
+    this.otherMethods = {
+      fioSDK: this.fioSDK,
+      walletLocalData: null,
+      walletLocalDataDirty: this.dirtyData,
+      fioAction(actionName, parameters): Promise<any> {
```

Comment:
We talked about this a bit in our last call. Breaking out individual methods adds quite a bit of overhead and are trying to avoid the need to have to update wallet engines every time we rev the sdk. We understand that it would be helpful to be able to know the explicit parameters, but the current generic method should limit the number of changes we have to implement when we update our calls (which may be in flux over the next months after mainnet).